x/globalfee/ante/fee_utils.go

```diff
@@ -31,6 +32,25 @@ func (mfd FeeDecorator) getGlobalFee(ctx sdk.Context, feeTx sdk.FeeTx) sdk.Coins
 	return requiredGlobalFees.Sort()
 }
 
+func (mfd FeeDecorator) DefaultZeroGlobalFee(ctx sdk.Context) []sdk.DecCoin {
+	bondDenom := mfd.getBondDenom(ctx)
+	// todo: check bonddenom is only one denom
+	if bondDenom == "" {
+		panic("staking bond denom is empty")
```

Comment:
Can we return an error and use `s.T().Fatalf(err)` in the parent method?

Reply:
Hi, you meant not panic in `DefaultZeroGlobalFee`, but return error `func (mfd FeeDecorator) DefaultZeroGlobalFee(ctx sdk.Context) ([]sdk.DecCoin, error)`, and then panic [here](https://github.com/cosmos/gaia/blob/b0b59461682a10c920646e65faed2c82d019f066/x/globalfee/ante/fee_utils.go#L21) if err ?

Reply:
yess!!! WDYT? This will be more clear to debug in the test logs